db/src/main/kotlin/edu/artic/db/AppDataManager.kt

```diff
@@ -5,34 +5,93 @@ import org.threeten.bp.LocalDateTime
 import org.threeten.bp.format.DateTimeFormatterBuilder
 import javax.inject.Inject
 
-class AppDataManager @Inject constructor(private val serviceProvider: AppDataServiceProvider) {
+class AppDataManager @Inject constructor(
+        private val serviceProvider: AppDataServiceProvider,
+        private val appDataPreferencesManager: AppDataPreferencesManager,
+        private val appDatabase: AppDatabase
+) {
+    companion object {
+        const val HEADER_LAST_MODIFIED = "last-modified"
+    }
 
     fun getBlob(): Observable<AppDataState> {
         return serviceProvider.getBlobHeaders()
                 .flatMap {
+
                     val dtf = DateTimeFormatterBuilder()
                             //Thu, 12 Jul 2018 16:56:12 GMT
                             .appendPattern("E, dd MMM yyyy HH:mm:ss O")
```

Comment:
pattern needs to go into a utility function in base or something.

Reply:
I'm thinking that if this pattern is used elsewhere we can, until then we can maybe keep it here?